loader/CMakeLists.txt

```diff
@@ -321,6 +321,18 @@ else()
         target_link_libraries(vulkan PRIVATE Threads::Threads)
     endif()
 
+    if (LOADER_ENABLE_ADDRESS_SANITIZER)
+        target_compile_options(vulkan PUBLIC -fsanitize=address)
+        target_link_options(vulkan PUBLIC -fsanitize=address)
+        if (LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING)
+            target_compile_definitions(vulkan PRIVATE LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING=1)
+        endif()
+    endif()
+    if (LOADER_ENABLE_THREAD_SANITIZER)
```

Comment:
I'd make this an `elseif` since you can't use both at the same time.

Reply:
I'd rather just have a warning about trying to enable both. `elseif` would work but would mean that if you *did* try to enable both, only asan would be active and so the user might think both are active (due to not getting any tsan errors).